### ISSUE TITLE
microk8s inspect does not detect  memory cgroup correctly when cgroupv2 is enabled

### ISSUE THREAD

**Issue by jbulow** [canonical/microk8s#3567]
#### Summary
When running `microk8s inspect` the memory cgroup is not detected
```
WARNING:  The memory cgroup is not enabled.
The cluster may not be functioning properly. Please ensure cgroups are enabled

```
#### What Should Happen Instead?
I assume microk8s should work with cgroupV2. 

#### Reproduction Steps

Environment:

```
debian@beaglebone:~$ lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye

debian@beaglebone:~$ uname -a
Linux beaglebone 5.10.145-ti-arm64-r72 #1xross SMP Tue Nov 15 09:03:52 CET 2022 aarch64 GNU/Linux

debian@beaglebone:~$ cat /proc/cmdline
cgroup_enable=memory cgroup_memory=1 console=ttyS2,115200n8 earlycon=ns16550a,mmio32,0x02800000 root=/dev/mmcblk0p2 ro rootfstype=ext4 rootwait net.ifnames=0 quiet
```
1. install snapd
2. install microk8s
3. run `microk8s inspect`

#### Introspection Report

[inspection-report-20221115_151516.tar.gz](https://github.com/canonical/microk8s/files/10013904/inspection-report-20221115_151516.tar.gz)

#### Are you interested in contributing with a fix?

Yes, if I had a fix.

**Comment by neoaggelos** [comment]
Hi @jbulow, thank you for raising the issue.

There was a recent fix for this with https://github.com/canonical/microk8s/pull/3518, but it has not yet been backported to all stable tracks. Can you see if installing MicroK8s from `latest/edge` does away with the error message?

Also, even though `microk8s inspect` prints this warning, if cgroup2 is in fact enabled, then containerd should be functioning properly without issues. Are you seeing any problems to spin up workloads/pods/containers/etc?

Thank you!

**Comment by dalbani** [comment]
I think this issue can be closed, now that MicroK8s v1.26 has been released with the fix.
